Makefile

```diff
@@ -379,7 +379,7 @@ test_no_lint:
 	cd repos/system_upgrade/el7toel8/; \
 	snactor workflow sanity-check ipu && \
 	cd - && \
-	$(_PYTHON_VENV) -m pytest $(REPORT_ARG) $(TEST_PATHS) $(LIBRARY_PATH) $(PYTEST_ARGS)
+	$(_PYTHON_VENV) -m pytest $(REPORT_ARG) $(LIBRARY_PATH) $(PYTEST_ARGS)
```

Comment:
good catch!

Reply:
Actually, I committed this by accident - if you do `make test_no_lint` you want to also test actors. I have removed the `$(TEST_PATHS)` because there was no way (at least obvious to me) to test just common libraries.

Reply:
I am actually puzzled now :-D because I checked the Makefile as the change looked weird to me originally but going through it, it started to make sense to me as a good step. Reading your explanation and the Makefile again, I am now trying to find out why I thought that  :D
 :face_in_clouds:  + :palm_tree:  -->  :facepalm:
 
 Even Homer nods :shrug: